client/src/api/AuthService.ts

```diff
@@ -0,0 +1,34 @@
+import axios, { AxiosResponse } from 'axios'
+import { IAxiosDataResponse } from './types'
+
+export default class AuthService {
+    static async validateUserToken(
+        token: string
+    ): Promise<AxiosResponse<IAxiosDataResponse>> {
+        const response = await axios.get<IAxiosDataResponse>(
+            'http://localhost:5000/api/auth/me',
```

Comment:
you should replace `http://localhost:5000` with the value from const and this const should be taken from a `.env` file. 

it means, at first you create a `config.js` file, which takes all the constants from appropriate .env file

Reply:
fix